shared/elements/gaia_lowstorage_dialog/examples/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
Yeah, this is correct. We probably want to create a folder to split the implementations. (I still think they should all just live inside gaia and following our conventions of underscores, but I seem to be losing that battle!)